cookbooks/cdo-apps/Berksfile

```diff
@@ -1,30 +1,12 @@
 source 'https://supermarket.chef.io'
 
 metadata
-%w(
-  repository
-  github-access
-  secrets
-  postfix
-  varnish
-  mysql
-  ruby
-  nginx
-  nodejs
-  java-7
-  networking
-  cloudwatch-extra-metrics
-  redis
-  solr
-  i18n
-).map {|x| "cdo-#{x}"}.
-  push('sudo-user').
-  each do |dep|
-  cookbook dep, path: "../#{dep}"
-end
+instance_eval File.read('../local_cookbooks.rb'), __FILE__
```

Comment:
This is just a little bit of refactoring, will help maintain this long list of cookbook-dependencies moving forward.